### DISCUSSION TITLE
path to .git

### DISCUSSION THREAD

**Discussion by m1roSlavit** [zed-industries/zed#8070] [Feature Requests]
I'm using an editor not from the root of my git repository. therefore the editor does not show git changes. How can I fix this?

**Comment by ghost** [comment]
I don't believe there is a workaround, at least a sane one. Most editors (zed included) checks the root directory of the opened project for the ``.git`` folder. I recommend opening the editor in the root of your git repository, even if that means you'd have two or more windows. VSCode does have workspaces, not sure how they work in specifics regarding detecting git, but that feature is missing in Zed as of now.

**Reply by Moshyfawn** [reply]
I think you are right that there is currently no way to prompt Zed to detect a git repository in the parent folders. However, this could potentially be added as a feature request if enough people work on projects in a similar way.

VSCode has implemented similar behaviour in the form of `git.repositoryScanMaxDepth` which works in conjunction with `git.autoRepositoryDetection` set to `true` or `subFolders` to detect `.git` in folders above or below the currently open folder.

https://code.visualstudio.com/docs/getstarted/settings#_default-settings

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
